### PR TITLE
change weapons while moving

### PR DESCRIPTION
**Pull request by commy2**

### CODE REVIEW

File: addons/weaponselect/XEH_postInit.sqf [hunk 1]
```diff
@@ -11,7 +11,9 @@ if (!hasInterface) exitWith {};
     if !([ACE_player] call EFUNC(common,canUseWeapon)) exitWith {false};
 
     // Statement
-    [ACE_player, handgunWeapon ACE_player] call FUNC(selectWeaponMode);
+    //[ACE_player, handgunWeapon ACE_player] call FUNC(selectWeaponMode);
+    [ACE_player, currentWeapon ACE_player] call FUNC(SelectPistolFast);
```
Comment (MikeMatrix) [inline addons/weaponselect/XEH_postInit.sqf]: Uppercase `s` and leftover code that's been commented out.

File: addons/weaponselect/XEH_postInit.sqf [hunk 2]
```diff
@@ -24,7 +26,8 @@ if (!hasInterface) exitWith {};
     if !([ACE_player] call EFUNC(common,canUseWeapon)) exitWith {false};
 
     // Statement
-    [ACE_player, primaryWeapon ACE_player] call FUNC(selectWeaponMode);
+    //[ACE_player, primaryWeapon ACE_player] call FUNC(selectWeaponMode);
+    [ACE_player, currentWeapon ACE_player] call FUNC(SelectRifleFast);
```
Comment (MikeMatrix) [inline addons/weaponselect/XEH_postInit.sqf]: Same here

File: addons/weaponselect/functions/fnc_selectRifleFast.sqf [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// by commy2
```
Comment (MikeMatrix) [inline addons/weaponselect/functions/fnc_selectRifleFast.sqf]: NO!

File: addons/weaponselect/functions/fnc_selectPistolFast.sqf [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// by commy2
```
Comment (MikeMatrix) [inline addons/weaponselect/functions/fnc_selectPistolFast.sqf]: NO!

File: addons/weaponselect/CfgMoves.hpp [hunk 5]
```diff
@@ -0,0 +1,107 @@
+
```
Comment (MikeMatrix) [inline addons/weaponselect/CfgMoves.hpp]: Empty line at the beginning of the file

File: addons/weaponselect/functions/fnc_selectPistolFast.sqf [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+#include "script_component.hpp"
+
+params ["_unit", "_weapon"];
+
+local _animIndex = [//@todo add anims here
+    "amovpercmstpsraswrfldnon",
+    "amovpercmrunsraswrfldf", "amovpercmrunsraswrfldb", "amovpercmrunsraswrfldl", "amovpercmrunsraswrfldr", "amovpercmrunsraswrfldbl", "amovpercmrunsraswrfldbr", "amovpercmrunsraswrfldfl", "amovpercmrunsraswrfldfr",
+    "amovpknlmrunsraswrfldf", "amovpknlmrunsraswrfldb", "amovpknlmrunsraswrfldl", "amovpknlmrunsraswrfldr", "amovpknlmrunsraswrfldbl", "amovpknlmrunsraswrfldbr", "amovpknlmrunsraswrfldfl", "amovpknlmrunsraswrfldfr"
+] find animationState _unit;
```
Comment (MikeMatrix) [inline addons/weaponselect/functions/fnc_selectPistolFast.sqf]: Is there a way to find these programmatically?

Comment (MikeMatrix) [inline addons/weaponselect/functions/fnc_selectPistolFast.sqf]: If not, we should definitely put them in a define/variable for this package. Don't repeat yourself.

File: addons/weaponselect/functions/fnc_selectRifleFast.sqf (hunk 3 above)
Comment (jokoho48) [inline addons/weaponselect/functions/fnc_selectRifleFast.sqf]: NOOOOOOO i will kill you Commy remove that next time if you open a PR

File: addons/weaponselect/functions/fnc_selectPistolFast.sqf (hunk 4 above)
Comment (jokoho48) [inline addons/weaponselect/functions/fnc_selectPistolFast.sqf]: NOOOOOOO i will kill you Commy remove that next time if you open a PR

**Pull request closed without merging**